### PR TITLE
Tweaks Radstorm event to be less impactful

### PR DESCRIPTION
**Pull request by Absolutely-Steph**
## About The Pull Request
This changes the radstorm event in the following ways:
Different look, easier to see through, purple specks/haze/distortion whatever
changes how it damages people, it now does radiation and burn damage
it does rad damage up to about 1k rads, then it begins the mutating/braindamaging(for IPCs)
I balanced it so that tanking the whole storm will give you:
50-150 burn
1000 rads
the storm radiates enough that it will hit the 1k rad limit around 80-90% of the storms elapsed time, resulting in mutations for the last 5-10 seconds(if you tanked the full storm).

## Why It's Good For The Game
radstorm bad(terrible), me no likey

## Changelog
:cl:
tweak: tweaked radstorms
balance: nerfed radstorms since they appear more often than in vanilla ss13, but are just as disruptive.
/:cl:

### CODE REVIEW

**Comment by Absolutely-Steph** [comment]
> Considering this changes almost the entire proc. You'd be better off making an override in a modular nsv file

I actually only changed like 4 lines, it says i changed the whole thing because i put it in an extra if statement and had to indent all of it.

I have no idea how any of the code works or why anything was done the way it is, the guy making it ought to have commented their code frankly.

**Comment by TheFakeElon** [comment]
Yeah I meant line diff wise. Which is all github cares about when it comes to conflicts

**Comment by covertcorvid** [comment]
The light static is a bit hard to notice, but not awful once you know about it. The burn damage is killing all the pets and damaging equipment such as the AI.

**Comment by TheFakeElon** [comment]
this should really be modularized but I digress

File: code/datums/weather/weather_types/radiation_storm.dm [hunk 1]
```diff
@@ -30,44 +30,46 @@
...
-							BRAIN_TRAUMA_SPECIAL = 5
-						) )
-						var/resistance = pick(
-							95;TRAUMA_RESILIENCE_BASIC,
-							// 30;TRAUMA_RESILIENCE_SURGERY,
-							// 15;TRAUMA_RESILIENCE_LOBOTOMY,
-							5;TRAUMA_RESILIENCE_MAGIC
-						)
-						H.gain_trauma_type( trauma_type, resistance )
+	if(L.radiation >= 1000)
```
Comment (Bokkiewokkie) [inline code/datums/weather/weather_types/radiation_storm.dm:33]:
```suggestion
	if(L.radiation >= 1000) //NSV start, puts a maximum on how irradiated you can get
```

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

File: code/datums/weather/weather_types/radiation_storm.dm (hunk 1 above)
Comment (Absolutely-Steph) [reply code/datums/weather/weather_types/radiation_storm.dm]: yeah that makes sense, good call. commited via the lil button it gave me here. super useful!

**Comment by Absolutely-Steph** [comment]
If this is no longer needed, i'd appreciate it closed so i can delete the branch please <3

**Pull request closed without merging**